src/GlobalStyles.js

```diff
@@ -17,4 +17,10 @@ export const GlobalStyles = createGlobalStyle`
     background: ${({ theme }) => theme.colors.Mercury};
     font-family: 'Poppins', sans-serif;
   }
+
+  #root {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
Ja mam tutaj pytanie, po co nam te style do #root? Dodatkowo czy dla mobilnego widoku height jako 100vh jest takie dobre? Zakomentowanie tego kawałka CSS i odpalenie testu nic nie zmienia

Reply:
Zmienia 😅 Jak się usunie z App.js paginację, której docelowo nie będzie na tej stronie tak jak pisałam w opisie, to bez tego fragmentu ten widok nie będzie zajmować całej wysokości strony, a powinien. 

Jeśli chodzi o 100vh i mobilki to niestety nie jestem znawcą :(  może masz rację, że lepiej to zastąpić czymś innym.

Reply:
+1 dla Kasi, sprawdzałem to.
100vh tylko dla tego komponentu załatwia sprawę, inaczej trzeba by było nadawać wysokość nadrzędnym jak body czy html,
i tak, zmienia :) polecam potestować :) 
szczególnie jak ta strona będzie osobna bez innych dodatkowych komponentów